src/ray/raylet/node_manager.cc

```diff
@@ -3272,6 +3311,15 @@ void NodeManager::HandleGetNodeStats(const rpc::GetNodeStatsRequest &request,
   }
 }
 
+void NodeManager::HandleGlobalGC(const rpc::GlobalGCRequest &request,
+                                 rpc::GlobalGCReply *reply,
+                                 rpc::SendReplyCallback send_reply_callback) {
+  RAY_LOG(WARNING) << "Broadcasting global GC request to all raylets.";
+  should_global_gc_ = true;
+  // We won't see our own request, so trigger local GC immediately too.
+  DoLocalGC();
```

Comment:
Could move this to `Heartbeat()`, or just set `should_local_gc_`, so that we also throttle if there are a bunch of global GC requests from all local workers.

Reply:
Fixed.